views/standard/spec/columns.js

```diff
@@ -293,6 +293,13 @@ define(['jquery', 'appearanceUtils', 'underscore'], function($, utils, _) {
                    label.created_at, label.user);
                   node.append(icon);
                }
+               if ((label = pull.getLabel('external_block'))) {
+                  var icon = $('<i>').addClass('fa fa-eye-slash externally-blocked');
```

Comment:
I'm not familiar with this syntax `$('<i>')`. What is it supposed to select?

Reply:
I thought it was invalid too, but I guess it's not a selector, rather creates a new `i` element and applies it to the pull later.

Reply:
Whoa, ok. TIL